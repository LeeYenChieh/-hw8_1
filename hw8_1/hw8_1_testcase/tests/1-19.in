MbeVFfb1paqX6bldz2YzAZsRO HQJL0KOwLqN0lLZB13JUQ4MB1mEOnu5nIorIFqn
PjYUSKuqtxWmDrW5uGbbs1h5t p8zWSSFwOd
oe1rl9N3 BUbSYVU9OWma3kEP3s1tgOyPJ2puaRODfP0fXHBrjL
25gslmRcy1 Otv3ZAwozMVyGFd6wbbVigRGuh
1MuZwkQr0QKliBqdmqRL0ZWMd XUaUPm
oe1rl9N3 display
0jxZrHD4EODTxRAJWWAaoIEXqroC4bqAPH4OFGCmVse10eUOxVLy BuLvBz6hkJd
1MuZwkQr0QKliBqdmqRL0ZWMd display
0jxZrHD4EODTxRAJWWAaoIEXqroC4bqAPH4OFGCmVse10eUOxVLy Eu431j6ZDIgmborQZX4rqyU4shV2wia5OAH
Pfr4zLOMjbStLtbXpWacMeyI9m eiCMr8B74IG5B
mmy9d3tImEEizY7JcLc83Q6UPKoAJrWE5qdjA8J0XAL7cf hooQFMRCj7MLIJ1
Pfr4zLOMjbStLtbXpWacMeyI9m q7stauzzBnnHM0teqDW5AuKwkC4TdQ
MbeVFfb1paqX6bldz2YzAZsRO display
1MuZwkQr0QKliBqdmqRL0ZWMd add scrollBar
gigFenZQS9rTeRqwTFak xU3mRuxD8z3TUtoDP
