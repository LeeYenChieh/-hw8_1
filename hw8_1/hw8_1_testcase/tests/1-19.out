BUbSYVU9OWma3kEP3s1tgOyPJ2puaRODfP0fXHBrjL 
XUaUPm 
HQJL0KOwLqN0lLZB13JUQ4MB1mEOnu5nIorIFqn 
